CHANGELOG/CHANGELOG-1.0.md

```diff
@@ -0,0 +1,102 @@
+# Changelog 1.0
```

Comment:
Question - should not be named CHANGELOG-1.x.md to be consistent with CHANGELOG-0.x.md?

Reply:
yes, @tnqn is correct, this is by design
I kept all releases pre-1.0 in the same file as these are past releases and there was no point in splitting the CHANGELOG at that point

Reply:
Then where should be put change logs for 1.x?

Reply:
`CHANGELOG/CHANGELOG-1.1.md`, `CHANGELOG/CHANGELOG-1.2.md`, etc
this is inspired by https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG
it makes it easier to update the CHANGELOG automatically and makes more sense because a CHANGELOG across multiple minor release trains is not really "linear"

Reply:
Ok. One more - when we have 2.0, do we merge 1.x to CHANGELOG-1.x.md?

Reply:
No, I'm not planning to do so. There will be `CHANGELOG-2.0.md`, `CHANGELOG-2.1.md`, etc
All releases pre-1.0 are in the same file because that's how it was before this reorg, and because automatic CHANGELOG updates won't apply to these releases

Reply:
Got it. That is my guess too. I am fine with the new way.